webrtc.html

```diff
@@ -2948,8 +2948,51 @@
             <code><a>RTCRtpReceiver</a></code> object.
             In the case where no a=mid attribute is present in the remote
             description, a random value will be generated.
-          </dd>
-        </dl>
+	  </dd>
+
+	  <dt> sequence&lt;RTCRtpContributingSource&gt;
+	    getContributingSources ()  </dt>
+	  <dd> Returns a RTCRtpContributingSource for each unique
+	  CSRC or SSRC received by this RTCRtpReceiver in the last 10
+	  seconds.  </dd>
+	  
+	</dl>
+
+	<p> The RTCRtpContributingSource objects contains information
+	information about a given contribution source, including the
+	time the most recent time a packet was received from the
+	source. The browser MUST keep information from RTP packets
+	received in the previous second. When a RTP packet is
+	received, the entries are updated for the SSRC of the RTP
+	packet and for any CSRC contained in it. </p>
+
+	 <dl class="idl" title="interface RTCRtpContributingSource">
+
+	   <dt> readonly attribute DOMHiResTimeStamp timestamp</dt>
+
+	   <dd> Time of reception of the most recent RTP packet
+	   containing the contributing source. </dd>
+
```

Comment:
the actual definition of what the timestamp refers to was discussed [on the list](https://lists.w3.org/Archives/Public/public-webrtc/2015Sep/0058.html); shouldn't this be reflected here? (either as "to be determined" or with a specific proposal for review)